README.md

```diff
@@ -19,7 +19,7 @@ $ sudo chmod a+x /usr/local/bin/symfony
 ### Windows
 
 ```bash
-c:\> php -r "readfile('http://symfony.com/installer');" > symfony
+c:\> php -r "file_put_contents('symfony', file_get_contents('http://symfony.com/installer'));"
```

Comment:
Opening URLs with the `file_get_contents()` function requires the `allow_url_fopen` INI setting to be `true`. Anyone know if that's the case for most Windows users?